src/App.js

```diff
@@ -1,17 +1,31 @@
 import "./App.css";
 import Navbar from "./components/navbar/header";
 import LandingPageBody from "./components/landing-page-body/landingPageBody";
-import { BrowserRouter, Route } from "react-router-dom";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import PhotosGallery from "./components/projects-gallary/projectsGallary";
 import Footer from "./components/footer/footer";
+import ExploreProjects from "./components/explore-projects/ExplorePojects";
 
 function App() {
   return (
     <BrowserRouter>
       <div className="App">
         <Navbar />
-        <PhotosGallery />
-        <LandingPageBody />
+
+        <Routes>
+          <Route
+            exact
+            path="/"
+            element={
+              <>
+                {" "}
+                <PhotosGallery />
+                <LandingPageBody />
+              </>
+            }
+          />
+          <Route path="/explore" element={<ExploreProjects />} />
```

Comment:
use `CacheRoute` from `react-router-cache-route` to cache this route instead of doing the same heavy calculation on every route hit

Reply:
I will create a new ticket to do this.